content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -425,7 +481,6 @@ Commerce (ICA) tax. The report can be found under :menuselection:`Accounting -->
 Colombian Statements --> Certificado de Retención en ICA`.
 
 .. image:: colombia/ica-report.png
```

Comment:
@samueljlieber Change to `retencion-ica-dian.png` already in [Google Drive folder](https://drive.google.com/drive/folders/11cXF1gXxwws_P9hUslnSJcfjheEbBJZN) :)